src/test/java/org/apache/commons/lang3/StringUtilsContainsTest.java

```diff
@@ -105,7 +105,7 @@ public void testContainsAny_StringCharArray() {
     }
 
     /**
-     * See http://www.oracle.com/technetwork/articles/javase/supplementary-142654.html
+     * See https://www.oracle.com/technetwork/articles/javase/supplementary-142654.html
```

Comment:
There is no such URL, it redirects to https://www.oracle.com/technical-resources/articles/javase/supplementary.html

Reply:
Done.